Libraries/LibJS/Runtime/Intl/NumberFormat.h

```diff
@@ -181,7 +182,7 @@ class NumberFormat final : public NumberFormatBase {
 
 int currency_digits(StringView currency);
 Vector<Unicode::NumberFormat::Partition> partition_number_pattern(NumberFormat const&, MathematicalValue const& number);
-String format_numeric(NumberFormat const&, MathematicalValue const& number);
+JS_API String format_numeric(NumberFormat const&, MathematicalValue const& number);
```

Comment:
Same question on some methods. `format_numeric` is only used in .cpp files within LibJS AFAICT.